tests/meta/orders/conftest.py

```diff
@@ -14,6 +14,7 @@
 from cg.meta.orders.ticket_handler import TicketHandler
 from cg.models.orders.order import OrderIn, OrderType
 from cg.store import Store
+from tests.apps.orderform.conftest import *
```

Comment:
I do not think it is a good idea to use `import *`. It will pollute the namespace

Reply:
Specified fixtures in import